fleet/lib/kube-prometheus-stack/aggregator/values.yaml

```diff
@@ -154,17 +152,42 @@ alertmanager:
           matchers:
             - receivers =~ ".*,squadcast,.*"
           continue: true
-        - receiver: slack-kube-test
+        - receiver: "slack-test"
```

Comment:
Note that on master the alertmanager configuration has been moved to overlays to seperate the test configs from the soon to be production configuration.  This PR needs to be rebased and presumably these changes moved to the ayekan overlay.